pyproject.toml

```diff
@@ -48,7 +48,7 @@ djangocms-blog = "1.1.1"
 djangocms-bootstrap4 = "2.0.0"
 djangocms-column = "1.11.0"
 djangocms-file = "3.0.0"
-djangocms-forms = "0.2.5"
+djangocms-forms-maintained = { git = "https://github.com/avryhof/djangocms-forms", rev = "ab38b2239b93c8a275cee3b4b1a7880731486c07" }
```

Comment:
**New Since https://github.com/TACC/Core-CMS/pull/500**
Get some [new commits](https://github.com/avryhof/djangocms-forms/commits/master), specifically:
- https://github.com/avryhof/djangocms-forms/issues/8
- https://github.com/avryhof/djangocms-forms/pull/12